components/ntp_background_images/browser/view_counter_model.cc

```diff
@@ -90,25 +90,14 @@ void ViewCounterModel::RegisterPageViewForBrandedImages() {
     // Reset count and randomize image index for next time.
     count_to_branded_wallpaper_ = kRegularCountToBrandedWallpaper;
 
-    if (always_show_branded_wallpaper_) {
```

Comment:
this is dead code because it's early return above when `always_show_branded_wallpaper_` is true.